modules/nlu/src/backend/engine.ts

```diff
@@ -418,7 +418,7 @@ export default class ScopedEngine implements Engine {
     }
 
     const intents = await this.intentClassifiers[ds.language].predict(
-      ds.tokens.map(t => t.sanitized),
+      ds.tokens.map(t => t.cannonical),
```

Comment:
Should'nt we pass the raw sentencepiece tokens here as discussed earlier ?

Reply:
out of the scope of this pr